src/pages/resultsView/plots/PlotsTab.tsx

```diff
@@ -165,6 +170,11 @@ export type PlotsTabDataSource = {
 };
 
 export type PlotsTabOption = { value: string; label: string };
+export type PlotsTabOptionWithGenericAssayInfo = {
```

Comment:
do we have to duplicate PlotsTabOption?   Seems like inheritanace would be good here.  Or just add it as meta node on PlotsTapOption

Reply:
@alisman Should I do 
```
type PlotsTabOptionWithGenericAssayInfo = PlotsTabOption & {isGenericAssayType?: boolean}
```

Reply:
@alisman I think I may just rewrite `PlotsTapOption` to this:
```
export type PlotsTabOption = {
    value: string;
    label: string;
    isGenericAssayType?: boolean;
};
```